jobs/config.json

```diff
@@ -2347,8 +2346,7 @@
     "--env-file=jobs/ci-kubernetes-soak-gce-federation-test.env",
     "--soak-test",
     "--up=false",
-    "--down=false",
-    "--tag=v20170223-43ce8f86"
```

Comment:
@krzyzacy I don't think anybody is paying attention to these tests right now. It is a total bloodbath. We need to fix that, so that people care about them. So I think it is fine to move the tag.